kubernetes/apps/network/external-dns/values.yaml

```diff
@@ -5,19 +5,21 @@ env:
   - name: CF_API_TOKEN
     valueFrom:
       secretKeyRef:
-        name: external-dns-secret
+        name: &secret external-dns-secret
         key: api-token
 extraArgs:
   - --cloudflare-dns-records-per-page=1000
   - --cloudflare-proxied
   - --crd-source-apiversion=externaldns.k8s.io/v1alpha1
   - --crd-source-kind=DNSEndpoint
-  - --events
   - --ignore-ingress-tls-spec
   - --ingress-class=external
+  - --gateway-name=external
+triggerLoopOnEvent: true
```

Comment:
this is an update to the helm chart that takes care of adding `--events` for you